docs/src/literate/quasi_incompressible_hyperelasticity.jl

```diff
@@ -26,7 +26,7 @@
 # ```math
 #     \Psi(\mathbf{u}) = \frac{\mu}{2}\left(I_1 - 3 \right) - \mu \log(J) + \frac{\lambda}{2}\left( J - 1\right){}^2,
 # ```
-# where $I_1 = \tf{\mathbf{C}} = F_{ij}F_{ij} = \tr(F)$ and $J = \det(\mathbf{F})$ denote the standard invariants of the deformation gradient tensor $\mathbf{F} = \mathbf{I}+\nabla \mathbf{u}$.
+# where $I_1 = \mathrm{tr}(\mathbf{C}) = \mathrm{tr}(F^T F) = F_{ij}F_{ij}$ and $J = \det(\mathbf{F})$ denote the standard invariants of the deformation gradient tensor $\mathbf{F} = \mathbf{I}+\nabla \mathbf{u}$.
```

Comment:
```suggestion
# where $I_1 = \mathrm{tr}(\mathbf{C}) = \mathrm{tr}(\mathbf{F}^\mathrm{T} \mathbf{F}) = F_{ij}F_{ij}$ and $J = \det(\mathbf{F})$ denote the standard invariants of the deformation gradient tensor $\mathbf{F} = \mathbf{I}+\nabla_{\mathbf{X}} \mathbf{u}$.
```
Just to be picky 😉, but otherwise looks good to me now! Nice doc improvement!

Reply:
Not picky, you are correct. :)